tyr/policies/ec2.py

```diff
@@ -211,8 +243,10 @@
         "sns:Subscribe"
       ],
       "Resource": [
-        "arn:aws:sns:us-east-1:761584570493:p-outpost",
-        "arn:aws:sns:us-east-1:761584570493:alyx3"
+        "arn:aws:sns:us-east-1:761584570493:prod-outpost",
```

Comment:
Can you remove this line?  It doesn't seem like test-outpost exists and you already have line for `t-outpost`.

```
t-outpost arn:aws:sns:us-east-1:761584570493:t-outpost
```